extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -183,17 +239,15 @@ export class ConsentPolicyManager {
    * @return {!Promise<boolean>}
    */
   whenPolicyUnblock(policyId) {
-    if (!ConsentPolicyManager.isMultiSupported(this.ampdoc_.win)) {
-      // If customized policy is not supported
-      if (!WHITELIST_POLICY[policyId]) {
-        user().error(TAG, 'can not find policy %s, ' +
-          'only predefined policies are supported', policyId);
-        return Promise.resolve(false);
-      }
+    // If customized policy is not supported
+    if (!WHITELIST_POLICY[policyId]) {
+      user().error(TAG, 'can not find policy %s, ' +
+        'only predefined policies are supported', policyId);
+      return Promise.resolve(false);
     }
     return this.whenPolicyInstanceReady_(policyId).then(() => {
       return this.instances_[policyId].getReadyPromise().then(() => {
-        return this.instances_[policyId].shouldBlock();
+        return this.instances_[policyId].shouldEnable();
```

Comment:
As we discussed offline, let's just keep this as unblock, my apologies 😢

Reply:
haha it's a great point! and I enjoy reading the article a lot! It's just blocking/unblocking the build of other AMP components sound better to me in this very specific use case.